MagickCore/stream.c

```diff
@@ -1290,8 +1290,15 @@ MagickExport Image *StreamImage(const ImageInfo *image_info,
   assert(stream_info->signature == MagickCoreSignature);
   assert(exception != (ExceptionInfo *) NULL);
   read_info=CloneImageInfo(image_info);
+  if (read_info == (ImageInfo *) NULL)
```

Comment:
`CloneImageInfo` will never return null because the first call is to `AcquireImageInfo` that will throw a fatal exception when it fails.

Reply:
You are right.  I reviewed the code, I don't know why i made this  mistake.